server/src/services/databaseOrm.py

```diff
@@ -43,40 +43,40 @@ class Ingest(Base):
     policeprecinct = Column(String)
 
 
-insertFields = {'srnumber': String,
+insertFields = {'srnumber': String(50),
```

Comment:
The padding makes me nervous (but I'm also not too DB smart) I almost wanna say we should set the length to the MySQL max instead?
OR 
Prove me wrong by ingesting all 5 years of data with these settings haha
OR
find some sorta of schema definition in mydatala to confirm the lengths

Reply:
OK I actually went ahead and ingested all 5 years worth of data (took 4.2 minutes!) and modified the varchars accordingly. making anothe rpull request soon